File: Programming-Basics-with-Python/9.While Loop - Lab/07. Min Number.py
# команда -> число под формата на текст или "Stop"
# повтарям: чета число + проверка дали е макс
# спирам: команда == "Stop"
# продължавам: команда != "Stop"
import sys
command = input()
min = sys.maxsize
while command != "Stop":
    # command -> число под формата на текст
    number = int(command)
    # проверка за минимум
    if number < min:
        min = number

    command = input()
else: # command == "Stop"
    print(min)

File: Programming-Basics-with-JavaScript/5.While Loop/07minNumber.js
function minNumber(input){
    let minNum = 999999999
    let index = 0
    let command = input[index]
    index++
    while (command != 'Stop'){
        number = Number(command)
        if (minNum > number){
            minNum = number

        }
        command = input[index]
        index++


    }
    console.log(minNum)



}
minNumber(["45",
"-20",
"7",
"99",
"Stop"])

